go.mod

```diff
@@ -232,3 +232,12 @@ replace github.com/johannesboyne/gofakes3 => github.com/chanwit/gofakes3 v0.0.0-
 // replace k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20220614142933-1062c7ade5f8
 
 // Use patched version that fixed recursive gets, and force delete for buckets
+
+//exclude gopkg.in/square/go-jose.v2 v2.6.0
+//exclude gopkg.in/square/go-jose.v2 v2.0.0-00010101000000-000000000000
+
+replace gopkg.in/square/go-jose.v2 => gopkg.in/go-jose/go-jose.v2 v2.6.3
```

Comment:
V2.6.3 is the API-compatible replacement without any other changes.

Reply:
This replacement is completely unnecessary because 2.6.3 is the latest version, anyway, and the one that would be picked by the go tooling. It can be removed.